x-pack/test/functional/services/observability/alerts/rules_page.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export function ObservabilityAlertsRulesProvider({ getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+
+  const getManageRulesPageHref = async () => {
+    const manageRulesPage = await testSubjects.find('manageRulesPages');
+    return manageRulesPage.getAttribute('href');
+  };
+
+  const clickCreateRuleButton = async () => {
+    const createRuleButton = await testSubjects.find('createRuleButton');
+    return createRuleButton.click();
+  };
+
+  const clickRuleStatusDropDownMenu = async () => testSubjects.click('statusContextButton');
+
+  const clickDisableFromDropDownMenu = async () =>
+    find.clickByCssSelector('div.euiContextMenuPanel > div > div > button:nth-child(2)');
```

Comment:
@fkanout Wasn't there an easier way to get access to the Disabled item? For example adding a `data-test-subj` to the dropdown options? If it doesn't work, using what you have is also fine by me

Reply:
I'm not a big fan of that selector either. I went with this as the list is created dynamically so `data-test-subj` should be too. So I used this one and it works.